server/public/styles.css

```diff
@@ -0,0 +1,20 @@
+* {
+  box-sizing: border-box;
+}
```

Comment:
This `box-sizing` reset is already included with Bootstrap.

Reply:
deleted